src/agents/onboarding_agent.py

```diff
@@ -223,14 +156,16 @@ def run(self, context: AgentContext) -> Action:  # noqa: C901
                 f"throughout the quest, and end with the player having either found an item that will "
                 f"help them in subsequent quests, or having achieved their ultimate goal.\n"
                 f"A player has requested a new game with the following attributes:\n"
-                f"Genre: {game_state.genre}\n"
-                f"Tone: {game_state.tone}\n"
+                f"Tone: {server_settings.narrative_tone}\n"
+                f"Background on the world of the story:\n"
+                f"{server_settings.adventure_background}"
                 f"The player is playing as a character named {game_state.player.name}. "
                 f"{game_state.player.name} has the following background: "
                 f"{game_state.player.background}\n"
-                f"{game_state.player.name}'s overall goal is to: {game_state.player.motivation}."
+                f"{game_state.player.name}'s overall goal is to: {server_settings.adventure_goal}."
```

Comment:
praise: i much prefer `adventure_goal` to `motivation`.

there is probably room for both, but this feels more focused.